pkg/auth/azure.go

```diff
@@ -101,11 +102,13 @@ func HttpAuthMiddleWare(resp http.ResponseWriter, req *http.Request, jwks *keyfu
 	token := req.Header.Get("authorization")
 	for _, allowedPath := range allowedPaths {
 		if req.URL.Path == allowedPath {
+			logger.FromContext(req.Context()).Info("path not allowed: " + req.URL.Path)
```

Comment:
The log message is the wrong way round isn't it?

Reply:
yes, removed